monitor/monitor.hpp

```diff
@@ -59,12 +76,18 @@ class monitor {
 
 	struct dnet_node *node() { return m_node; }
 
+	typedef std::shared_ptr<top_stats> top_stats_ptr;
+	top_stats_ptr get_top_stats() const { return m_top_stats; }
+
 private:
 	struct dnet_node	*m_node;
 	server		m_server;
 	statistics	m_statistics;
+	top_stats_ptr m_top_stats;
```

Comment:
`m_top_stats` is a part of all statistics and should be moved into `m_statistics` with `get_top_stats()`.